media-gfx/qrencode/qrencode-3.4.4.recipe

```diff
@@ -40,6 +40,7 @@ REQUIRES_devel="
 	"
 
 BUILD_REQUIRES="
+	qrencode_kdl
```

Comment:
@localanu, I've just checked on x86_gcc2 that qrencode builds fine without this change.
I might be wrong, but I guess there is no reason to add qrencode_kdl to haikuports/media-gfx/qrencode/qrencode-3.4.4.recipe.
Did you notice a problem with that recipe?

Reply:
@fbrosson in x86_64 ,I found this problem : 
Warning: POLICY ERROR: no matching provides "devel:libqrencode_kdl" for "develop/lib/libqrencode_kdl.a"

Reply:
You need to remove the qrencode_kdl. It's not needed.

Reply:
in x86 is not needed,but in x86_64?

Reply:
qrencode_kdl exists to allow a user to "export" messages displayed in Kernel Debug Land (if I'm not mistaken) by using QRcodes and then taking pictures with a camera.
My understanding is that no package should depend on qrencode_kdl. All "allowed" functionality (to users outside the kernel) is already provided by the qrencode package.